src/blocks/modal.css

```diff
@@ -102,8 +102,8 @@
 }
 
 .modal__image {
-  width: 100%;
-  height: 100%;
+  width: 80vw;
+  height: 80vh;
```

Comment:
You should add `max-width` and `max-height` rather than just `width` and `height`, otherwise the `close icon` and `caption` is not correctly positioned https://snipboard.io/1eAiP9.jpg


```css
.modal__image {
  max-width: 80vw;
  max-height: 80vh;
  object-fit: contain;
}

```